### PR TITLE
modelize_class: Remove a dead check

### PR DESCRIPTION
**Pull request by jcbrinfo**
Dead at aa64312 (PR #2047).

Signed-off-by: Jean-Christophe Beaupré <jcbrinfo@users.noreply.github.com>

### CODE REVIEW

File: src/modelize/modelize_class.nit [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # This file is part of NIT ( http://www.nitlanguage.org ).
 #
-# Copyright 2012 Jean Privat <jean@pryen.org>
+# Copyright 2012-2016 Jean Privat <jean@pryen.org>
+# Copyright 2015      Alexandre Terrasa <alexandre@moz-code.org>
+# Copyright 2016      Jean-Christophe Beaupré <jcbrinfo@users.noreply.qithub.com>
```
Comment (privat) [inline src/modelize/modelize_class.nit]: please do not update Copyrights

File: src/modelize/modelize_class.nit [hunk 2]
```diff
@@ -104,11 +106,6 @@ redef class ModelBuilder
 		end
 
 		if mclass == null then
-			if nclassdef isa AStdClassdef and nclassdef.n_kwredef != null then
-				error(nclassdef, "Redef Error: no imported class `{name}` to refine.")
-				return
-			end
```
Comment (privat) [inline src/modelize/modelize_class.nit]: good catch

**Comment by jcbrinfo** [comment]
:recycle: Without copyright notice

**Comment by privat** [comment]
+1

**Pull request merged**